desktop/release-notes.md

```diff
@@ -24,12 +24,53 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.11.1
+2022-08-05
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed `docker login` to private registries when Registry Access Management is enabled and access to Docker Hub is blocked [docker/for-win#12871](https://github.com/docker/for-win/issues/12871)
```

Comment:
```suggestion
- Fixed `docker login` to private registries from WSL2 distro [docker/for-win#12871](https://github.com/docker/for-win/issues/12871)
```

Reply:
Thanks for that, does this mean it's a windows-only issue @fredericdalleau?

Reply:
I don't know tbh, I've gone only through one diagnostic, and reproduce only on wsl2 distro (and it's a for-win) so it's a reasonable bet.